src/Controllers/SignUpController.php

```diff
@@ -101,6 +102,12 @@ private function renderSignUpPage(): Response
                 'isPronounEnabled' => $this->config->get('enable_pronoun'),
                 'isFullNameEnabled' => $this->config->get('enable_user_name'),
                 'isPlannedArrivalDateEnabled' => $this->config->get('enable_planned_arrival'),
+                'isPronounRequired' => $requiredFields['pronoun'],
```

Comment:
Using variables here imho does not make it better readable but as it done so before thats ok (and should definitely be refactored at a later stage)

Reply:
I like it that way!
won't change